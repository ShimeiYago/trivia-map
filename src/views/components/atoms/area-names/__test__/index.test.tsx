import { shallow, ShallowWrapper } from 'enzyme';
import { AreaNames, Props } from '..';

let wrapper: ShallowWrapper<Props>;

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(
      <AreaNames
        areaNames={['シー', 'メディテレーニアンハーバー', 'ポルトパラディーゾ']}
      />,
    );
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
