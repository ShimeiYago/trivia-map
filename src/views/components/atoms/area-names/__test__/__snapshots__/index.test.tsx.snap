// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<ForwardRef(Typography)
  sx={
    Object {
      "alignItems": "center",
      "columnGap": undefined,
      "display": "flex",
      "flexWrap": "wrap",
      "fontSize": undefined,
      "justifyContent": "left",
    }
  }
>
  シー
  <Memo(ForwardRef(KeyboardArrowRightIcon))
    fontSize="inherit"
  />
  メディテレーニアンハーバー
  <Memo(ForwardRef(KeyboardArrowRightIcon))
    fontSize="inherit"
  />
  ポルトパラディーゾ
</ForwardRef(Typography)>
`;
